docs/who_are_we.md

```diff
@@ -0,0 +1,39 @@
+
+# Who Are We ?
+
+## Foreword
+
+We are excited to count among our ranks numerous key contributors to so many key project of the Python Open Source Ecosystem.
+So many companies / groups / organizations who recognized the value of fixing the issues WheelNext is trying to approach.
+
+Below you will find the list of communities, projects and companies involved in making WheelNext great !
+
+## Open-Source Projects (alphabetic order)
+
+- AstroPy: https://www.astropy.org/
+- Conda: <https://docs.conda.io/projects/conda/en/stable/>
+- GPU-Mode: <https://github.com/gpu-mode>
+- JAX: <https://docs.jax.dev/en/latest/>
+- NumFocus: <https://numfocus.org/>
+- NumPy: <https://numpy.org/>
+- Numba: <https://numba.pydata.org/>
+- PyTorch: <https://pytorch.org/>
+- RAPIDS: <https://rapids.ai/>
+- Scikit-Build: https://scikit-build.readthedocs.io/en/latest/
+- Scikit-Learn: https://scikit-learn.org/
+- Triton: <https://openai.com/index/triton/>
+- UV: <https://docs.astral.sh/uv/>
```

Comment:
```suggestion
- uv: <https://docs.astral.sh/uv/>
```

Reply:
(Noted I'll correct that in a further commit immediately after this one is merged - I don't want to void the approvals here)